service_container/service_subscribers_locators.rst

```diff
@@ -140,6 +140,18 @@ count and iterate over the services of the locator::
     The :class:`Symfony\\Contracts\\Service\\ServiceCollectionInterface` was
     introduced in Symfony 7.1.
 
+Without autowire
```

Comment:
maybe this?
```suggestion
Manually wiring a locator
```

Reply:
or a `.. note::` indeed as suggested by Oskar

Reply:
is it possible to add code block in a note ?